polyply/src/nonbond_engine.py

```diff
@@ -81,6 +81,22 @@ def __init__(self,
                  interaction_matrix,
                  cut_off,
                  boxsize):
+        """
+        Parameters:
+        -----------
+        positions: np.ndarray
+        nodes_to_idx: dict[(mol_idx, node_key)]
+             correspondance dict between indices in position array
+             and a unique node in the system identified by molecule
+             index and node_key
+        atomtypes:  np.ndarray
+             array of atom_types corresponding to the atoms in positions
+        interaction_matrix: dict[forzenset(atom_type, atom_type)]
```

Comment:
typo
And what are the dict values?

Reply:
strings defining the atom_type. Thought that is quite obvious.

Reply:
```
interaction_matrix; dict[frozenset(str, str), tuple(float, float)]
    Dict mapping a set of atomnames to a tuple of containing ....
```
That the atom_ytpes are str is fairly obvious. What's unclear is what it actually maps to.